src/mod.ts

```diff
@@ -28,6 +29,82 @@ export const run = async function (functionDir: string) {
   console.log(JSON.stringify(resp || {}));
 };
 
+// Start a http server that listens on the provided port
+// This server exposes two routes
+//  GET  `/health`    Returns a 200 OK
+//  POST `/functions` Accepts event payload and invokes `run`
+const startServer = async function (port: number) {
```

Comment:
Most of this is right out of the deno samples https://deno.land/manual@v1.15.0/runtime/http_server_apis